router/router.go

```diff
@@ -0,0 +1,23 @@
+package router
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+// RootRouter is router that binds passed handler functions to the root path
+func RootRouter(handler http.HandlerFunc) {
+	http.HandleFunc("/", handler)
+
+	port := os.Getenv("PORT")
```

Comment:
Tell me whyyyyyyyy!
once you assign value to variable 'port' outside if, and later, you do assigment to variable in if body, this 'err' variable

Reply:
initial assignment ment in the if statement has scope limited to if statement body. I need the port variable after if body for ListenAndServe func so I can't assign port as an initial assignment in its if. I don't need err outside its if body tho, so I can assign it inline as initial variable

Reply:
Fine... xD